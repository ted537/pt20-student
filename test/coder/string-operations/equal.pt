using output;
if ("hello"=="hello") then
	put("hello equals hello");
	putln;
else
	put("uh oh");
	putln;
end;
if ("red"=="blue") then
	put("uh oh");
	putln;
else
	put("red does not equal blue");
	putln;
end;
