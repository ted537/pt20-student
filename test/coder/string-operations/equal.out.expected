hello equals hello
red does not equal blue
